iothub/service/src/RegistryManager.cs

```diff
@@ -70,48 +70,30 @@ public class RegistryManager : IDisposable
         private IHttpClientHelper _httpClientHelper;
 
         /// <summary>
-        /// Creates an instance of <see cref="RegistryManager"/>, provided for unit testing purposes only.
```

Comment:
No need to self-ref to the same topic we're already on.

Reply:
The only benefit of self-ref would be to get consistency if we ever rename this class. hint hint

Reply:
Meh, cost of user confusion is worse IMO.